src/View.Uno/View.Uno.csproj

```diff
@@ -1,19 +1,20 @@
-﻿<Project Sdk="MSBuild.Sdk.Extras/2.0.54">
+﻿<Project Sdk="MSBuild.Sdk.Extras/3.0.44">
   <PropertyGroup>
-		<!-- Change the TargetFrameworks depending on which platform you are building on. This avoids errors as it is impossible to build UAP on OSX (MacOS) -->
-		<TargetFrameworks Condition="'$([MSBuild]::IsOsPlatform(OSX))'">netstandard2.0;xamarinios10;</TargetFrameworks>
-		<TargetFrameworks Condition="'!$([MSBuild]::IsOsPlatform(OSX))'">netstandard2.0;xamarinios10;monoandroid11.0;monoandroid12.0;uap10.0.19041</TargetFrameworks>
-		<!-- Ensures the .xr.xml files are generated in a proper layout folder -->
-		<GenerateLibraryLayout>true</GenerateLibraryLayout>
-		<RootNamespace>Nventive.View</RootNamespace>
-		<Authors>nventive</Authors>
-		<Company>nventive</Company>
-		<AssemblyName>Nventive.View.Uno</AssemblyName>
-		<PackageId>Nventive.View.Uno</PackageId>
-		<Description>Nventive.View.Uno</Description>
-		<GenerateDocumentationFile>true</GenerateDocumentationFile>
-		<_Uno_XamlMerge_Task_Version>1.1.0-dev.13</_Uno_XamlMerge_Task_Version>
-		<XamlMergeOutputFile>$(MSBuildProjectDirectory)\Themes\MergedResourceDictionaries.xaml</XamlMergeOutputFile>
+	<!-- Change the TargetFrameworks depending on which platform you are building on. This avoids errors as it is impossible to build UAP on OSX (MacOS) -->
+	<TargetFrameworks Condition="'$([MSBuild]::IsOsPlatform(OSX))'">netstandard2.0;xamarinios10;</TargetFrameworks>
+	<TargetFrameworks Condition="'!$([MSBuild]::IsOsPlatform(OSX))'">netstandard2.0;xamarinios10;monoandroid11.0;monoandroid12.0;uap10.0.19041</TargetFrameworks>
+	<!-- Ensures the .xr.xml files are generated in a proper layout folder -->
+	<GenerateLibraryLayout>true</GenerateLibraryLayout>
+	<RootNamespace>Nventive.View</RootNamespace>
+	<Authors>nventive</Authors>
+	<Company>nventive</Company>
+	<AssemblyName>Nventive.View.Uno</AssemblyName>
+	<PackageId>Nventive.View.Uno</PackageId>
+	<Description>Nventive.View.Uno</Description>
+	<GenerateDocumentationFile>true</GenerateDocumentationFile>
+	<PackageTags>mvvm;xamarin;ios;android;uwp;chinook;maui;winui;</PackageTags>
```

Comment:
remove chinook, maui and winui

Reply:
We have this in other packages, it was an addition that was asked even in the Uno project https://github.com/nventive/Chinook.DynamicMvvm/pull/71#discussion_r975552630
https://github.com/nventive/Chinook.DynamicMvvm/pull/71#discussion_r975674744

Reply:
Uno.csproj is not maui and winui project - the comment is in a WinUI project

Reply:
The second comment says to add maui and winui to all csproj which was done since MessageDialog. So if we go this route, just to be consistent, we should do the same for all packages that already have been updated

Reply:
Most of the uno csproj don't have packagetags, I have PRs on the way to clean the projects and add consistency